apps/condo/domains/organization/integrations/sbbol/utils/changeClientSecret.js

```diff
@@ -2,16 +2,20 @@ const querystring = require('querystring')
 const dayjs = require('dayjs')
 const conf = require('@open-condo/config')
 const { SbbolRequestApi } = require('../SbbolRequestApi')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
 const { getOrganizationAccessToken } = require('./getOrganizationAccessToken')
 const { getSbbolSecretStorage } = require('./getSbbolSecretStorage')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
 
 const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
 const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
 
-async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret }) {
-    const accessToken = await getOrganizationAccessToken()
+async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret, userId }) {
     const sbbolSecretStorage = getSbbolSecretStorage()
-
+    const { keystone: userContext } = await getSchemaCtx('User')
+    const user = User.getOne(userContext, { importRemoteSystem: SBBOL_IMPORT_NAME, deletedAt: null })
+    const accessToken = userId ? await getOrganizationAccessToken(userId) : await getOrganizationAccessToken(user.id)
```

Comment:
Could you please write comment about such strange substitution of accessToken of some first-hand user, since SBBOL API is unaware of that and requires just some `accessToken` that is not obligatory related to user of organization that has integration at SBBOL side.